will/backends/io_adapters/slack.py

```diff
@@ -347,7 +356,7 @@ def send_message(self, event):
         })
         if hasattr(event, "kwargs") and "html" in event.kwargs and event.kwargs["html"]:
             data.update({
-                "parse": "full",
+                "parse": "none",
```

Comment:
What's the intent behind this change?

Reply:
When the Will converts html to Slack markdown, we need to change the parse to "none" so Slack doesn't perform any processing in the message. SlackConverter is in charge of the markdown and formatting.

> If you don't want Slack to perform any processing on your message, pass an argument of parse=none.
> IN  : Foo <!everyone> bar http://test.com
> OUT : Foo <!everyone> bar http://test.com
> (In this case, Slack will still test the validity of your markup - we won't send invalid messages to clients).

https://api.slack.com/docs/message-formatting#message_formatting